### PR TITLE
Fix: Fixed issue where add item was disabled in column layout

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   1. Fixed issue where "add item" action was disabled in secondary columns.
   2. Fixed issue where "add item' was disabled when items were selected (it should still be hidden in the right click menu)

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open a second column in the columns layout
   3. Confirm the new/add item action is enabled

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Actions/FileSystem/AddItemAction.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal class AddItemAction : ObservableObject, IAction
 
 		public RichGlyph Glyph { get; } = new(opacityStyle: "ColorIconNew");
 
-		public bool IsExecutable => context.CanCreateItem && !context.HasSelection;
```
Comment (hishitetsu) [inline src/Files.App/Actions/FileSystem/AddItemAction.cs:27]: This change affects more than just column layout. Are there any regression concerns with removing this? I don't know why the action is disabled while items are selected though.

Comment (yaira2) [reply src/Files.App/Actions/FileSystem/AddItemAction.cs:27]: That's right, it also fixes an issue where the option was disabled when items were selected.

Comment (hishitetsu) [reply src/Files.App/Actions/FileSystem/AddItemAction.cs:27]: @ferrariofilippo why did you add `!context.HasSelection` to `IsExecutable`?
https://github.com/files-community/Files/commit/f52fda7b09b2f197431e406150e7571fb3fa7697

Comment (yaira2) [reply src/Files.App/Actions/FileSystem/AddItemAction.cs:27]: I assume it was to hide the option when right clicking on multiple files. However, this is handled in the context menu helper so it's okay to remove it from the action.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**